src/_check.sh

```diff
@@ -128,3 +128,38 @@ function _check_base_required_packages() {
   fi
   return $status
 }
+
+function check_etl_status() {
+  title "Checking CB->PG ETL status"
+
+  RAW_OUTPUT=$(`compose_client exec plextracapi npm run pg:etl:status`)
```

Comment:
This npm script has not been merged to main yet. I'm going to leave this PR in draft until I get that script merged into the backend repo first.